mailpoet/RoboFile.php

```diff
@@ -479,7 +479,6 @@ public function qaLintBuild() {
       'lib-3rd-party/',
       'vendor/composer',
       'vendor/mtdowling',
-      'vendor/soundasleep',
```

Comment:
Checking the commit message, it is not clear to me why it was necessary to include `vendor/soundasleep` in the PHP linter check, and now that it was moved to vendor prefixed, it is not necessary anymore. Could you please explain? In my opinion, this is the kind of information that is useful to include in the commit message.

Reply:
Just for the records: I'm going to add this line back.